packages/analytics-browser/src/config/joined-config.ts

```diff
@@ -31,6 +32,8 @@ export class BrowserJoinedConfigGenerator {
       this.config.defaultTracking = remoteConfig.defaultTracking;
     }
     this.config.loggerProvider.debug('Joined configuration: ', JSON.stringify(remoteConfig, null, 2));
+    this.config.requestMetadata ??= new RequestMetadata();
```

Comment:
Does this only track fetch time when the remote config is actually fetched?

Reply:
This is implemented in analytics-remote-config package. I'm collecting the time of both fetching from remote and from indexDB currently because both of them takes time and would potentially affect amplitude.init()

Reply:
I mean I am hoping we don't send 0s to the backend as that may heavily influence the average time it takes to fetch data from the CDN

Reply:
It won't be 0 because either fetching from remote or from indexedDB will take some time.